playbooks/common/roles/remove-home-extend-root/tasks/main.yaml

```
@@ -4,6 +4,7 @@
       - '!all'
       - hardware
 - include_tasks: remove_home_partition.yaml
+  when: 'remove_home_partition == True'
 - include_tasks: extend_root_partition.yaml
-  when: "extend_root_partition == 'true'"
+  when: 'extend_root_partition == True'
```

Do we need explicit `...|bool` here?
```suggestion
  when: 'extend_root_partition|bool == True'
```
And if yes, what happens when `extend_root_partition` value is some string like `hello world`?